modules/localgov_subsites_paragraphs/localgov_subsites_paragraphs.module

```diff
@@ -143,7 +143,7 @@ function localgov_subsites_paragraphs_preprocess_paragraph(&$variables) {
       $button = $paragraph->get('localgov_button')->first()->getValue();
       $variables['button_url'] = Url::fromUri($button['uri']);
     }
-    if (!$paragraph->get('localgov_background_image')->isEmpty()) {
+    if ($paragraph->get('localgov_background_image')->entity) {
```

Comment:
This isn't the first time this section has caused us pain.
https://github.com/localgovdrupal/localgov_paragraphs/pull/125

Feels like it's brittle and needs tests. There is however nothing and this will fix the specific issue.